packages/core/helper-plugin/lib/src/components/ReactSelect/ReactSelect.js

```diff
@@ -11,7 +11,7 @@ import getSelectStyles from './utils/getSelectStyles';
 
 const ReactSelect = ({ components, styles, ...props }) => {
   const theme = useTheme();
-  const customStyles = getSelectStyles(theme);
+  const customStyles = getSelectStyles(theme, props);
```

Comment:
@gu-stav didn't find another way to access error and applying style but not sure it's the best, do you have another solution in mind?